### DISCUSSION TITLE
apollo修改密码后，忘记密码怎么办

### DISCUSSION THREAD

**Discussion by 631994449** [apolloconfig/apollo#4194] [Q&A]
因需求修改了apollo账号的登录密码。但不知为何某天后无法登录了。
现在user表内有一个apollo用户和user用户，通过sql将apollo密码修改为user的密码，也无法用user的密码登录apollo用户。
能否有办法找回apollo用户。

**Comment by lepdou** [comment]
recreate apollo user?

> INSERT INTO `Users` (`Username`, `Password`, `UserDisplayName`, `Email`, `Enabled`)
VALUES
	('apollo', '$2a$10$7r20uS.BQ9uBpf3Baj3uQOZvMVvB1RN3PYoKE94gtz2.WAOuiiwXS', 'apollo', 'apollo@acme.com', 1);

**Reply by 631994449** [reply]
delete apollo user and recteate apollo user. still can not login with "apollo/admin"

**Reply by 13602517521** [reply]
是的，还是无法登录，portal无报错日志

**Comment by nobodyiam** [comment]
There was a related bug, see #2975.
